### PR TITLE
Noel's globe demo

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: demos/globe/third-party/Three/Detector.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/**
```
Comment (dstoc) [inline demos/globe/third-party/Three/Detector.js:1]: Should we add this as a git submodule?

File: demos/globe/third-party/twine.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/*
```
Comment (dstoc) [inline demos/globe/third-party/twine.js]: Missing header.

Comment (dstoc) [inline demos/globe/third-party/twine.js]: Is this meant to be in third-party?

**Comment by dstoc** [comment]
This seems to contain a copy of web-animations.js at demos/globe/third-party/animations.js

File: demos/globe/third-party/twine.js (hunk 2 above)
Comment (shans) [inline demos/globe/third-party/twine.js]: yes. The .tween.html version of the demo uses it.

File: demos/globe/third-party/Three/Detector.js (hunk 1 above)
Comment (shans) [inline demos/globe/third-party/Three/Detector.js:1]: I'd prefer this to be standalone as much as possible.

**Comment by shans** [comment]
PTAnotherL

**Comment by dstoc** [comment]
index.html and twine.js need apache license

**Comment by shans** [comment]
again? :)

File: demos/globe/index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
+<!--
+Copyright 2012 Google Inc. All Rights Reserved.
```
Comment (dstoc) [inline demos/globe/index.html]: It's 2013 :) (same in twine.js)

**Comment by dstoc** [comment]
lgtm

**Pull request merged**